src/server/main_service.cc

```diff
@@ -1260,6 +1260,10 @@ void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx)
     dfly_cntx->transaction->ScheduleSingleHopT(cb);
   }
 
+  if (cntx->conn()) {
```

Comment:
I would say that based on the code above ` cntx->conn()` must always be true. is not it the case?

Reply:
No, not on ClusterMode, it was crashing here when it tried to deference `conn`